system/modules/core/library/Contao/Encryption.php

```diff
@@ -47,9 +47,14 @@ class Encryption
 	 * @param string $strKey   An optional encryption key
 	 *
 	 * @return string The encrypted value
+	 *
+	 * @deprecated Deprecated since Contao 3.5, to be removed in Contao 5.
```

Comment:
This message feels really wrong to me. Please **NEVER** implement your own encryption routine but use existing and tested libraries for this :-) The message as it is now implies you should invent something on your own :-D